modules/admin-ui-frontend/app/scripts/shared/directives/datetimepickerDirective.js

```diff
@@ -61,9 +61,14 @@ angular.module('adminNg.directives')
         };
 
 
-        var optionsObj = {};
-        optionsObj.timeInput = true;
-        optionsObj.showButtonPanel = true;
+        var optionsObj = {
+          controlType: 'select',
+          showMillisec: false,
+          showMicrosec: false,
+          showTimezone: false,
+          oneLine: true,
+          timeFormat: 'HH:mm'
+        }
```

Comment:
The pipeline fails because of the missing semicolon.

Reply:
Yep, seems like the old eslint version in 10.x has slightly different rules about this.
But it's already fixed.